pkg/kubelet/kubelet.go

```diff
@@ -1041,6 +1055,14 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, dockerContainers dockertools.Docke
 			glog.Errorf("Failed to introspect pod infra container: %v; Skipping pod %q", err, podFullName)
 			return err
 		}
+		if kl.networkPluginName != "" {
+			// call the networking plugin
+			out, err := exec.New().Command(kl.networkPluginName, string(PostPodInfraLaunch), pod.Name, pod.Namespace, string(podInfraContainerID)).CombinedOutput()
```

Comment:
I was thinking of plugin names more as the basename of the path to the executable.  Something like

```
plugins := enumeratePlugins("/usr/local/bin/kubelet-plugins/net")
...
p := plugins[kl.networkPluginName]
if p != nil {
    p.PostLaunch()
}
```

That gives us the ability to check that the plugin exists early on, and to have linked-in and external plugins.  It's a bit more complex, but I think it's a nicer API.  Something like the pkg/kubelet/volume/plugins pattern, but with an Exec module that enumerates a given dir (or dirs) for new plugins.